source/shared/net/desktop-wallet/single-sign-schedule.rst

```diff
@@ -30,7 +30,7 @@ schedule and an explicit schedule.
 
 #. Go to **Accounts** and select the relevant account.
 
-#. Select **More**, and then select **Send GTU with a schedule**
+#. Select **Change view**, and then select **Send GTU with a schedule**
```

Comment:
A reflection again: This doesn't feel very obvious, when it has to be explained. Probably fine for now, but I guess something more specific than "Change view" would be good.